formbuilder/components/builder/JsonSchemaDownloader.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import btoa from "btoa";
+
+export default function JsonSchemaDownloader(props) {
+  const filename = props.schema.title + ".json";
+  const fileContent = "data:text/plain;base64," + btoa(JSON.stringify(props.schema));
```

Comment:
As commented in #144 I think we should use `application/json` rather than `text/plain` here.